src/System.Reflection.Context/tests/CustomReflectionContextTests.cs

```diff
@@ -9,6 +9,7 @@ namespace System.Reflection.Context
     public class CustomReflectionContextTests
     {
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
```

Comment:
I assume the behavior on desktop is something more robust and we're just missing tests for it?